torch/distributed/_shard/sharded_tensor/shard.py

```diff
@@ -17,11 +20,18 @@ class Shard:
         metadata(:class `torch.distributed._shard.sharded_tensor.ShardMetadata`):
             The metadata for the shard, including offsets, lengths and device placement.
     """
+
     __slots__ = ['tensor', 'metadata']
     tensor: torch.Tensor
     metadata: ShardMetadata
 
     def __post_init__(self):
+        """Verification for a shard.
```

Comment:
Please remove the new dosctrings and only fix the errors listed in the issue. I think the issues clearly asks to skip the "Missing docstring" errors.